avro/src/ser.rs

```diff
@@ -691,11 +694,13 @@ mod tests {
             a: 27,
             b: "foo".to_owned(),
             decimal: Decimal::from(vec![1, 24]),
+            big_decimal: BigDecimal::new(BigInt::from(12), 2),
         };
         let expected = Value::Record(vec![
             ("a".to_owned(), Value::Long(27)),
             ("b".to_owned(), Value::String("foo".to_owned())),
             ("decimal".to_owned(), Value::Bytes(vec![1, 24])),
+            ("big_decimal".to_owned(), Value::String("0.12".into())),
```

Comment:
The deserialization tests can in some cases use a different Value type than what is expected i the serialization test, e.g. for decimal/big_decimal. Maybe one should add tests that verifies a serialization-deserialization roundtrip: struct = fromValue(toValue(struct)) (It works now because both Value::String and Value::BigDecimal ends up calling the `visit_str` on BigDecimal)